src/demos/shell/demo-app-advanced-center-row.html

```diff
@@ -343,7 +343,7 @@ <h2>Boomer v Millennial - California</h2>
                 <tr>
                   <td class="sml"><calcite-icon icon="square" scale="s"></calcite-icon></td>
                   <td>01</td>
-                  <td>Can't keep Johnny down.</td>
+                  <td>LAST</td>
```

Comment:
Nitpick: if this is meant to call out being last, can you update this to "This should be last" or something similar?

Reply:
👍